app/presenters/hyrax/admin/repository_object_presenter.rb

```diff
@@ -18,13 +18,17 @@ def as_json(*)
         delegate :blacklight_config, to: CatalogController
 
         def counts
-          translation = { 'false' => :published, 'true' => :unpublished, nil => :unknown }
+          translation = translation_keys
           raw_count = Hash[*results.to_a.flatten]
           @counts ||= raw_count.each_with_object({}) { |(k, v), o| o[translation[k]] = v }
         end
 
         def search_builder
-          Stats::WorkStatusSearchBuilder.new(self)
+          if @object_type == 'works'
```

Comment:
I'm seeing this check for object_type == `works` 3 or so times in this class. Which makes me wonder if it should be a constant in case the name of this default gets changed in the future.